src/renderer/scss/component/_card.scss

```diff
@@ -283,6 +282,12 @@
   &:last-of-type {
     padding-bottom: $spacing-vertical * 2/3;
   }
+
+  // For the message if community top bids are disabled
+  .card__content.help {
```

Comment:
This is needed because of the special CSS that happens inside the CategoryList component, that allows content to scroll "off" the side of the edge. That behavior doesn't exist anywhere else in the app.